### PR TITLE
1.7.6.4 Release Note

### PR DESCRIPTION
**Pull request by MatShir**

### CODE REVIEW

File: news/_posts/releases/2020-03-04-prestashop-1-7-6-4-maintenance-release.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+![1.7.6.4 is available!](/assets/images/2020/03/1.7.6.4_banner.jpg)
+
+This maintenance release fixes 9 issues reported on the versions between 1.7.6.0 to the 1.7.6.3.
+Since these versions fix security problems, we **strongly** encourage you to upgrade your shop as soon as possible.
+We suggest upgrading your shop quickly to benefit from these fixes. Of course, don’t forget to backup before.
+
+The security update fixes protect your shop for attackers to change your customer's personal information.
+
+Reminder:  the 1-Click Upgrade module’s latest version is v[VERSION], don’t forget to upgrade it.
```
Comment (matks) [inline news/_posts/releases/2020-03-04-prestashop-1-7-6-4-maintenance-release.md]: Dont forget adding version

File: news/_posts/releases/2020-03-04-prestashop-1-7-6-4-maintenance-release.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title:  "Release Of PrestaShop 1.7.6.4"
+subtitle: "MAINTENANCE VERSION OF THE 1.7.6.X BRANCH"
+date:   2019-11-28 16:00:00
```
Comment (ttoine) [inline news/_posts/releases/2020-03-04-prestashop-1-7-6-4-maintenance-release.md]:
```suggestion
date:   2020-03-04 10:00:00
```

File: news/_posts/releases/2020-03-04-prestashop-1-7-6-4-maintenance-release.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+authors: [ PrestaShop ]
+image: /assets/images/2019/11/meta-1762.png
```
Comment (matks) [inline news/_posts/releases/2020-03-04-prestashop-1-7-6-4-maintenance-release.md]: 1762 or 1764 ?

File: news/_posts/releases/2020-03-04-prestashop-1-7-6-4-maintenance-release.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+* Display issues on the SEO live previews on Safari desktop ([#17525](https://github.com/PrestaShop/PrestaShop/issues/17525))
+* A combination color wasn’t displayed when it was selected ([#17437](https://github.com/PrestaShop/PrestaShop/issues/17437))
+* Couldn’t view a category if all subcategories were disabled ([#17379](https://github.com/PrestaShop/PrestaShop/issues/17379))
+* Some products combination wasn’t displayed when depending on the stock and the product configuration([#17345](https://github.com/PrestaShop/PrestaShop/issues/17345)) 
+* Support link was not displayed properly in the order confirmation in Arabic ([#17242](https://github.com/PrestaShop/PrestaShop/issues/17242))
+* Email’s subject can have special character again  ([#16847](https://github.com/PrestaShop/PrestaShop/issues/16847))
+* A reduction error occurred on the invoice when applying specific cart rule to an order 
+([#16491](https://github.com/PrestaShop/PrestaShop/issues/16491))
+
+Read the [full changelog here](https://github.com/PrestaShop/PrestaShop/releases/tag/1.7.6.4).
```
Comment (ttoine) [inline news/_posts/releases/2020-03-04-prestashop-1-7-6-4-maintenance-release.md:43]:
```suggestion
Read the [full changelog here](https://github.com/PrestaShop/PrestaShop/releases/tag/1.7.6.4).


```

**Review by ttoine — CHANGES_REQUESTED** [review]

**Review by matthieu-rolland — APPROVED** [review]

File: news/_posts/releases/2020-03-04-prestashop-1-7-6-4-maintenance-release.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+icon: icon-checked
+tags:
+ - 1.7
+ - 1.7.6.x
+ - version
+ - patch
+ - releases
+---
+
+After a few months dedicated to fixing a security issue and all regressions found on the previous 1.7.6 versions, PrestaShop 1.7.6.4 is finally available!
```
Comment (colinegin) [inline news/_posts/releases/2020-03-04-prestashop-1-7-6-4-maintenance-release.md]: After a few weeks

File: news/_posts/releases/2020-03-04-prestashop-1-7-6-4-maintenance-release.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+ - version
+ - patch
+ - releases
+---
+
+After a few months dedicated to fixing a security issue and all regressions found on the previous 1.7.6 versions, PrestaShop 1.7.6.4 is finally available! 
+
+![1.7.6.4 is available!](/assets/images/2020/03/1.7.6.4_banner.jpg)
+
+This maintenance release fixes 9 issues reported on the versions between 1.7.6.0 to the 1.7.6.3.
```
Comment (colinegin) [inline news/_posts/releases/2020-03-04-prestashop-1-7-6-4-maintenance-release.md]: reported from versions 1.7.6.0 to 1.7.6.3.

File: news/_posts/releases/2020-03-04-prestashop-1-7-6-4-maintenance-release.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+Since this version fixes a security problem, we **strongly** encourage you to upgrade your shop as soon as possible.
+We suggest upgrading your shop quickly to benefit from these fixes. Of course, don’t forget to backup before.
+
+The security patch protects your shop against attacks aimed at modifying your customers' personal information.
+
+Reminder:  the 1-Click Upgrade module’s latest version is v4.10.1, don’t forget to upgrade it.
+
+## Main fixes
+Below are listed some of the main regression issues we fixed in this version :
```
Comment (colinegin) [inline news/_posts/releases/2020-03-04-prestashop-1-7-6-4-maintenance-release.md]: Below are listed all regression issues

File: news/_posts/releases/2020-03-04-prestashop-1-7-6-4-maintenance-release.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+* An error case on the customer email validation ([#17809](https://github.com/PrestaShop/PrestaShop/issues/17809))
+* Failed load the product_page.css and translations.css files ([#17584](https://github.com/PrestaShop/PrestaShop/issues/17584))
```
Comment (colinegin) [inline news/_posts/releases/2020-03-04-prestashop-1-7-6-4-maintenance-release.md]: Failure on loading

File: news/_posts/releases/2020-03-04-prestashop-1-7-6-4-maintenance-release.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+* Display issues on the SEO live previews on Safari desktop ([#17525](https://github.com/PrestaShop/PrestaShop/issues/17525))
+* A combination color wasn’t displayed when it was selected ([#17437](https://github.com/PrestaShop/PrestaShop/issues/17437))
+* Couldn’t view a category if all subcategories were disabled ([#17379](https://github.com/PrestaShop/PrestaShop/issues/17379))
```
Comment (colinegin) [inline news/_posts/releases/2020-03-04-prestashop-1-7-6-4-maintenance-release.md]: View on category not possible when all subcategories were disabled

File: news/_posts/releases/2020-03-04-prestashop-1-7-6-4-maintenance-release.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+* Some products combination wasn’t displayed depending on the stock and the product configuration ([#17345](https://github.com/PrestaShop/PrestaShop/issues/17345))
```
Comment (colinegin) [inline news/_posts/releases/2020-03-04-prestashop-1-7-6-4-maintenance-release.md]: Some product combinations were not displayed on particular stock and product configurations

File: news/_posts/releases/2020-03-04-prestashop-1-7-6-4-maintenance-release.md [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+
+* An error case on the customer email validation ([#17809](https://github.com/PrestaShop/PrestaShop/issues/17809))
+* Failed load the product_page.css and translations.css files ([#17584](https://github.com/PrestaShop/PrestaShop/issues/17584))
+* Display issues on the SEO live previews on Safari desktop ([#17525](https://github.com/PrestaShop/PrestaShop/issues/17525))
+* A combination color wasn’t displayed when it was selected ([#17437](https://github.com/PrestaShop/PrestaShop/issues/17437))
+* Couldn’t view a category if all subcategories were disabled ([#17379](https://github.com/PrestaShop/PrestaShop/issues/17379))
+* Some products combination wasn’t displayed depending on the stock and the product configuration ([#17345](https://github.com/PrestaShop/PrestaShop/issues/17345)) 
+* Support link was not displayed properly in the order confirmation in Arabic ([#17242](https://github.com/PrestaShop/PrestaShop/issues/17242))
+* Email’s subject can have special character again ([#16847](https://github.com/PrestaShop/PrestaShop/issues/16847))
+* A reduction error occurred on the invoice when applying specific cart rule to an order
```
Comment (colinegin) [inline news/_posts/releases/2020-03-04-prestashop-1-7-6-4-maintenance-release.md]: An error on the discount occurred on the invoice when applying a specific cart rule to an order

File: news/_posts/releases/2020-03-04-prestashop-1-7-6-4-maintenance-release.md [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+* Support link was not displayed properly in the order confirmation in Arabic ([#17242](https://github.com/PrestaShop/PrestaShop/issues/17242))
+* Email’s subject can have special character again ([#16847](https://github.com/PrestaShop/PrestaShop/issues/16847))
+* A reduction error occurred on the invoice when applying specific cart rule to an order 
+([#16491](https://github.com/PrestaShop/PrestaShop/issues/16491))
+
+Read the [full changelog here](https://github.com/PrestaShop/PrestaShop/releases/tag/1.7.6.4).
+
+
+## Important changes 
+Below are the security information we added in this version:
```
Comment (colinegin) [inline news/_posts/releases/2020-03-04-prestashop-1-7-6-4-maintenance-release.md]: Find below detailed information about the security fix provided in this version:

File: news/_posts/releases/2020-03-04-prestashop-1-7-6-4-maintenance-release.md [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+Below are listed some of the main regression issues we fixed in this version :
+
+* An error case on the customer email validation ([#17809](https://github.com/PrestaShop/PrestaShop/issues/17809))
+* Failed load the product_page.css and translations.css files ([#17584](https://github.com/PrestaShop/PrestaShop/issues/17584))
+* Display issues on the SEO live previews on Safari desktop ([#17525](https://github.com/PrestaShop/PrestaShop/issues/17525))
+* A combination color wasn’t displayed when it was selected ([#17437](https://github.com/PrestaShop/PrestaShop/issues/17437))
+* Couldn’t view a category if all subcategories were disabled ([#17379](https://github.com/PrestaShop/PrestaShop/issues/17379))
+* Some products combination wasn’t displayed depending on the stock and the product configuration ([#17345](https://github.com/PrestaShop/PrestaShop/issues/17345)) 
+* Support link was not displayed properly in the order confirmation in Arabic ([#17242](https://github.com/PrestaShop/PrestaShop/issues/17242))
+* Email’s subject can have special character again ([#16847](https://github.com/PrestaShop/PrestaShop/issues/16847))
```
Comment (colinegin) [inline news/_posts/releases/2020-03-04-prestashop-1-7-6-4-maintenance-release.md]: Display issue on emails' subject with special characters

**Review by colinegin — CHANGES_REQUESTED** [review]

**Comment by MatShir** [comment]
Final review @ttoine before merging. We are ready!

**Pull request merged**